Formula/stubby.rb

```diff
@@ -0,0 +1,84 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://getdnsapi.net/blog/dns-privacy-daemon-stubby/"
+  url "https://getdnsapi.net/releases/getdns-1-1-1/getdns-1.1.1.tar.gz"
+  sha256 "fa414c30d5f2d2b2453b5cec77362b4cc0f44d440be5893233748d82bd6a1a56"
+
+  head do
+    url "https://github.com/getdnsapi/getdns.git", :branch => "develop"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on "openssl"
+  depends_on "libevent" => :recommended
+  depends_on "libev" => :optional
+  depends_on "libuv" => :optional
+
+  def install
+    if build.head?
+      system "glibtoolize", "-ci"
+      system "autoreconf", "-fiv"
+    end
+
+    args = [
+      "--with-ssl=#{Formula["openssl"].opt_prefix}",
+      "--with-trust-anchor=#{etc}/getdns-root.key",
+    ]
+    args << "--with-libevent" if build.with? "libevent"
+    args << "--with-libev" if build.with? "libev"
+    args << "--with-libuv" if build.with? "libuv"
+
+    # Current Makefile layout prevents simultaneous job execution
+    # https://github.com/getdnsapi/getdns/issues/166
+    ENV.deparallelize
```

Comment:
OK - just talked with one of the developers. Have re-opened that issue and a fix is now in develop. Will update the formula when the next release is available.